charts/images.yaml

```diff
@@ -53,6 +53,15 @@ images:
   repository: k8s.gcr.io/kube-proxy
   targetVersion: ">= 1.17"
 - name: hyperkube # used for kubectl + kubelet binaries on the worker nodes
+- name: etcd-backup-restore
+  sourceRepository: github.com/gardener/etcd-backup-restore
+  repository: eu.gcr.io/gardener-project/gardener/etcdbrctl
+  tag: "0.8.0-dev-dd26b65a2ab4efd77805584b33e8c559dd3c6d53"
+- name: etcd-druid
+  sourceRepository: github.com/gardener/etcd-druid
+  repository: eu.gcr.io/gardener-project/gardener/etcd-druid
+  tag: "0.1.0-dev"	
```

Comment:
Just for reminder, while merging we should first make the release of etcd-druid and update the tag here.

Reply:
Done